dpe/src/context.rs

```diff
@@ -65,6 +65,7 @@ impl<C: Crypto> Context<C> {
         self.handle = *args.handle;
         self.tci = TciNodeData::new();
         self.tci.tci_type = args.tci_type;
+        self.tci.locality = args.locality;
```

Comment:
I think we also need to update DeriveChild and InitializeContext to set the locality properly

Reply:
Oh, we're already doing the right things in both cases. I though locality was a new argument to active. Thanks!